modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/api/ItSqlSynchronousApiTest.java

```diff
@@ -131,4 +140,101 @@ public void process(ResultSet<SqlRow> resultSet) {
             resultSet.forEachRemaining(res::add);
         }
     }
+
+    @Test
+    public void testEarlyQueryTimeout() {
```

Comment:
these tests should be added and reworked accordingly to base ItSqlApiBaseTest class to test different APIs

Reply:
Fixed.